scripts/bundleSize/bundleSizeConfig.js

```diff
@@ -1,10 +1,13 @@
 module.exports = {
-  // Size limit for all bundles used by each service (K)
-  //
-  // Keep the MAX_SIZE +5 above the largest value and MIN_SIZE -5
-  // below the smallest value in the build output; this avoids the
-  // need for frequent changes as bundle sizes fluctuate.
+  /** Size limit for all bundles used by each service (kB)
+   *
+   * `MIN_SIZE` should be the value of "Smallest total bundle size (kB) (smallest service + smallest page)" &
+   * `MAX_SIZE` should be the value of "Largest total bundle size (kB) (largest service + largest page)"
+   * from the "MODERN service config & theme + page bundle sizes summary" in the build output
+   *
+   * We are allowing a variance of -5 on `MIN_SIZE` and +5 on `MAX_SIZE` to avoid the need for frequent changes, as bundle sizes can fluctuate
+   */
 
-  MIN_SIZE: 766,
-  MAX_SIZE: 1220,
+  MIN_SIZE: 802 - 5,
+  MAX_SIZE: 1237 + 5,
```

Comment:
Seems like quite a jump in min/max sizes from these changes. Or did this creep up from elsewhere but was just caught here?